crates/turborepo-auth/src/auth/logout.rs

```diff
@@ -1,5 +1,31 @@
-use turborepo_ui::{cprintln, GREY, UI};
+use tracing::error;
+use turborepo_api_client::Client;
+use turborepo_ui::{cprintln, GREY};
 
-pub fn logout(ui: &UI) {
-    cprintln!(ui, GREY, ">>> Logged out");
+use crate::{Error, LogoutOptions};
+
+pub fn logout<T: Client>(options: &LogoutOptions<T>) -> Result<(), Error> {
+    if let Err(err) = remove_token(options) {
+        error!("could not logout. Something went wrong: {}", err);
+        return Err(err);
+    }
+
+    cprintln!(options.ui, GREY, ">>> Logged out");
+    Ok(())
+}
+
+fn remove_token<T: Client>(options: &LogoutOptions<T>) -> Result<(), Error> {
+    let global_config_path = options.path;
+    let content = global_config_path.read_to_string()?;
+
+    let mut data: serde_json::Value = serde_json::from_str(&content)?;
+    if let Some(token) = data.get_mut("token") {
+        // Set token field to empty string
+        *token = "".into();
```

Comment:
I'm pretty sure we want `null` and not an empty string
```suggestion
        *token = Value::Null;
```

Reply:
Wasn't 100% sure how all this interacts with `null` but in theory it's fine

Reply:
> Wasn't 100% sure how all this interacts with null but in theory it's fine

That sounds like we need more testing.